cmd/release-controller/http.go

```diff
@@ -699,6 +701,8 @@ func (c *Controller) httpReleaseInfo(w http.ResponseWriter, req *http.Request) {
 		return
 	}
 
+	fmt.Fprintf(w, "<div class=\"alert alert-warning\">This site is part of OpenShift's continuous delivery pipeline. Neither the builds linked here nor the upgrade paths tested here are officially supported. Please visit <a href=\"https://access.redhat.com/downloads/content/290/\">the Red Hat Customer Portal</a> for the latest supported product details.</div>")
```

Comment:
I do not think this should be part of every page.  I'm also not sure presenting it as a visually disruptive alert is appropriate.

Reply:
We're seeing more and more people both internal and external referencing these pages as if they're authoritative for OCP. I don't know the UI framework here, is there something one step down from alert? I just worry not calling attention to it will lead to it going un-noticed.

Reply:
Maybe we call attention to it for the next 30 or 90 days and then demote it?

Reply:
The yellow box was the same "warning" box that we use elsewhere on the site.  I elected against the "alert", red, box because I figured it was too much.

Reply:
How about this?
![RC8](https://user-images.githubusercontent.com/43015045/95509894-1d304300-0983-11eb-85d8-ddfce5c7d700.png)

Reply:
That looks good to me, lets see what Clayton says.

Reply:
I have updated the code to utilize the blue box only on the 3 primary pages (accessible by the "Quick Links").